<launch>

	<node pkg="obstacles_detector" type="QR_code_detector_node" name="QR_code_detector_nodee" output="screen">
		
		<remap from="input_image" to="/vrep/vision_sensor"/>
		<remap from="output_image" to="/detector/image_raw"/>
		
	</node>

 	<node name="image_view" pkg="image_view" type="image_view" >
     	<remap from="image" to="/vrep/vision_sensor"/>
 	</node>

</launch>
